src/SerializableAsync.ts

```diff
@@ -160,10 +218,24 @@ export class SerializableAsync extends SerializableBase implements ISerializable
         return await this.postFrom(realObj)
     }
 
+    /**
+     * `preFrom` can be overwritten to manipulate the value before the parsing.
+     * This allows to add logic to the deserialization without having to override the fromAny method.
```

Comment:
This allows to add logic to the **parsing** without having to override the fromAny method.